integration-tests/smoke/log_poller_test.go

```diff
@@ -97,6 +98,8 @@ func executeBasicLogPollerTest(t *testing.T) {
 	lpTestEnv := prepareEnvironment(l, t, &testConfig)
 	testEnv := lpTestEnv.testEnv
 
+	ctx := context.Background()
```

Comment:
Is it possible to use testutils context or something similar here?

Reply:
Actually you can use: `testcontext.Get(t)`

Reply:
Sounds good